slave-nodejs6-centos7/test/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+#
+# Test the Jenkins image.
+#
+# IMAGE_NAME specifies the name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+docker run --rm -t --entrypoint=/bin/bash ${IMAGE_NAME} -ic 'source /usr/local/bin/configure-slave && oc version'
```

Comment:
I'm sourcing the configure-slave file explicitly here, since we're overriding the entrypoint, so it doesn't get called by run-jnlp-client.

Reply:
ok